index.html

```diff
@@ -983,25 +996,25 @@ <h2>
       </h2>
 
       <p>
-For the broadest interoperability, <a>DID</a> normalization should be as
-simple and universal as possible. Therefore:
+For the broadest interoperability, <a>DID</a> normalization should be as simple
```

Comment:
```suggestion
For the broadest interoperability, <a>DID</a> normalization is as simple
```

Reply:
How about "...ensure DID normalization is as simple..."?

Reply:
My thinking was, in trying to dodge that non-normative should, that this is the spec where we say what the normalization rules are so we can just authoritatively state that it is simple and universal. But I don't feel strongly, could also leave it as is.

Reply:
Ah. OK, I see where you are coming from now. Reworded as
"For the broadest interoperability, make DID normalization as simple and universal as possible:"